### ISSUE TITLE
Use versioning for grammar

### ISSUE THREAD

**Issue by szarnyasg** [slizaa/slizaa-opencypher-xtext#47]
I'd like to suggest the use of versioning for the grammar - instead of keeping it at `1.0.0-SNAPSHOT`. I think a versioning scheme that incorporates the openCypher version would be useful.
(Our incremental query engine project, [ingraph](https://github.com/FTSRG/ingraph) depends on the Slizaa grammar and uses it for compiling the queries.)

**Comment by wuetherich** [comment]
So what exactly is your suggestion? Version 10.0.0.SNAPSHOT for the current M10 branch and 10.0.0 for an official release (see https://semver.org/)?

**Comment by szarnyasg** [comment]
Good question. Applying semver for grammars is pretty difficult, because even the smallest changes, such as https://github.com/slizaa/slizaa-opencypher-xtext/pull/41/files, break the API and would necessitate a new major version. So it doesn't really translate well and I'd recommend deviating from it.

The [openCypher versioning scheme](https://s3.amazonaws.com/artifacts.opencypher.org/website/ocig5/openCypher+Versioning.pdf#page=3) denotes the current version as Cypher 9. (Cypher 10 will support multiple graphs, compositionality, and subqueries, while Cypher 11 will introduce path patterns.)
One way to incorporate this into the Slizaa versioning is to use the Cypher version as the major version (incrementing this has the most "breaking" changes), and use the milestone number as the minor version. For example, the grammar on `opencypher_spec_M10` branch would have a version of  `9.10.0-SNAPSHOT`. Small changes, such as https://github.com/slizaa/slizaa-opencypher-xtext/pull/41/files, could bump the patch version (which is sort of appropriate - while this changes the API, it actually only breaks compatibility for a few users).

**Comment by wuetherich** [comment]
I think your suggested solution fits best here... so I already changed the version to 9.10.0-SNAPSHOT on the current spec_M10 branch. I also merged the spec_M10 branch back to the master branch (the 'old' master is now on branch 'abandoned_master' ).

For future releases, the implementation of the latest opencypher version should take place on the master branch. Fixes for prior releases can be done on the corresponding branch. 

As always: comments and suggestion are highly appreciated ;-)
